src/controller.rs

```diff
@@ -127,12 +132,15 @@ pub fn start(
                                             "{} language server doesn't support method {}",
                                             ctx.language_id, method
                                         ),
+                                        ErrorCode::ServerError(CONTENT_MODIFIED) => String::new(),
```

Comment:
It might be better to reorg code to not use msg emptiness for control flow, something like this:

```rust
                                    match failure.error.code {
                                        ErrorCode::ServerError(CONTENT_MODIFIED) => {}
                                        code => {
                                            let msg = match code {
                                                ErrorCode::MethodNotFound => format!(
                                                    "{} language server doesn't support method {}",
                                                    ctx.language_id, method
                                                ),
                                                _ => format!(
                                                    "{} language server error: {}",
                                                    ctx.language_id, editor_quote(&failure.error.message)
                                                ),
                                            };
                                            ctx.exec(meta, format!("lsp-show-error {}", editor_quote(&msg)));
                                        }
                                    };

```

This also would leave the handling of ` ErrorCode::ServerError(CONTENT_MODIFIED)` extensible in case we will want to log or retry it in the future.

Reply:
Thanks for the quick feedback!